lib/game_team.rb

```diff
@@ -1,35 +1,35 @@
 class GameTeam
-  attr_reader :game_id, 
-              :team_id, 
-              :home_or_away, 
-              :game_result, 
-              :reg_or_ot, 
-              :coach, 
-              :goals, 
-              :shots, 
-              :tackles, 
-              :pim, 
-              :power_play_opp, 
-              :power_play_goals, 
-              :face_off_win_percentage, 
-              :giveaways, 
+  attr_reader :game_id,
+              :team_id,
+              :home_or_away,
+              :game_result,
+              :reg_or_ot,
+              :coach,
+              :goals,
+              :shots,
+              :tackles,
+              :pim,
+              :power_play_opp,
+              :power_play_goals,
+              :face_off_win_percentage,
+              :giveaways,
               :takeaways
 
   def initialize(data)
     @game_id = data[:game_id]
     @team_id = data[:team_id]
-    @home_or_away = data[:hoa]
+    @home_or_away = data[:HoA]
```

Comment:
About the capital letters: I thought the header conversion to symbol made everything lower case? My file is working currently with all lower case characters in these keys.

Reply:
yes it converts to lowercase